doc/help/commands.asciidoc

```diff
@@ -892,13 +894,27 @@ How many tabs to switch forward.
 
 [[tab-only]]
 === tab-only
-Syntax: +:tab-only [*--prev*] [*--next*]+
+Syntax: +:tab-only [*--prev*] [*--next*] [*--force*]+
 
 Close all tabs except for the current one.
 
 ==== optional arguments
 * +*-p*+, +*--prev*+: Keep tabs before the current.
 * +*-n*+, +*--next*+: Keep tabs after the current.
+* +*-f*+, +*--force*+: Avoid confirmation for pinned tabs.
+
+[[tab-pin]]
+=== tab-pin
+Syntax: +:tab-pin+
+
+Pin the current/[count]th tab.
+
+==== count
+The tab index to pin
+
+==== note
+* Pinning a tab shrinks it to `tabs->pinned-width` size.
+* Attempting to close a pinned tab will cause a confirmation, unless `--force` is passed.
```

Comment:
I should have probably figured that out when I realized this was all busywork... I'll regenerate the docs and make sure it all looks good (and add docstrings if needed) :).